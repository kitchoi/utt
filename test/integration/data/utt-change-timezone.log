2018-08-08 07:50+0900 hello
2018-08-08 11:20+0900 work

2018-08-08 11:02+0100 traveling