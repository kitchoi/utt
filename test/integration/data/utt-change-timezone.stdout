
---------------------- Wednesday, Aug 08, 2018 (week 32) -----------------------

Working Time: 10h02 [11h12]
Break   Time: 0h00 [0h00]

----------------------------------- Projects -----------------------------------

(10h02) : traveling, work

---------------------------------- Activities ----------------------------------

(7h42) : traveling
(2h20) : work


----------------------------------- Details ------------------------------------

(2h20) 00:00-11:20 work
(7h42) 11:20-11:02 traveling

